appveyor.yml

```diff
@@ -2,72 +2,138 @@
 version: '{branch}-{build}'
 skip_tags: true
 max_jobs: 1
-image: Visual Studio 2017
-init:
-  - git config --global core.autocrlf input
-# Uncomment the following line to show RDP info at beginning of job
-#  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
-clone_folder: c:\projects\mixxx
+image:
+  - Visual Studio 2017
+  - Ubuntu1804
+
+configuration:
+  - release-fastbuild
+  - release
+#  - debug
+
+platform:
+# Disable x86 builds since we only get one concurrent build on AppVeyor and x86
+# failures will be caught by Jenkins.
+#  - x86
+  - x64
+
+
+matrix:
+  exclude:
+    # Ubuntu doesn't support "release-fastbuild".
+    - image: Ubuntu1804
+      configuration: release-fastbuild
+    # We only want "release-fastbuild" for Windows since "release" consumes too
+    # much memory due to link-time code generation / whole-program optimization.
+    - image: Visual Studio 2017
+      configuration: release
```

Comment:
According to the comment, this is backwards

Reply:
This is within an "exclude" block, it's specifying what part of the matrix to drop.